packages/api/src/schema/schema.yaml

```diff
@@ -1498,46 +1498,72 @@ components:
-              type: object
+          oneOf:
+            - type: object
               additionalProperties: false
               required:
-                - accessKeyId
-                - secretAccessKey
-              description: Credentials for the output video storage
+                - type
+                - endpoint
+                - bucket
+                - credentials
+              description: Storage for the output files
               properties:
-                accessKeyId:
+                type:
+                  type: string
+                  enum:
+                    - s3
+                  description: Type of service used for output files
+                endpoint:
+                  type: string
+                  format: uri
+                  pattern: ^http(s)?://
+                  description: Service endpoint URL
+                  example: https://gateway.storjshare.io
+                bucket:
                   type: string
-                  description: Access Key ID
-                  example: AKIAIOSFODNN7EXAMPLE
-                secretAccessKey:
+                  description: Bucket with output files
+                  example: outputbucket
+                credentials:
+                  type: object
+                  additionalProperties: false
+                  required:
+                    - accessKeyId
+                    - secretAccessKey
+                  description: Credentials for the output video storage
+                  properties:
+                    accessKeyId:
+                      type: string
+                      description: Access Key ID
+                      example: AKIAIOSFODNN7EXAMPLE
+                    secretAccessKey:
+                      type: string
+                      description: Secret Access Key
+                      example: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+            - type: object
+              additionalProperties: false
+              required:
+                - type
+                - credentials
+              description: Storage for the output files
+              properties:
+                type:
                   type: string
-                  description: Secret Access Key
-                  example: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+                  enum:
+                    - web3.storage
+                  description: Type of service used for output files
+                credentials:
+                  type: object
+                  additionalProperties: false
+                  required:
+                    - proof
+                  description:
+                    Delegation proof for Livepeer to be able to upload to
+                    web3.storage
+                  properties:
+                    proof:
+                      type: string
+                      description: Base64 encoded UCAN delegation proof
+                      example: did:key:z6MkkwKy2C9sCgoWcCGgxVCCt1qFyLJzwkoWi9BF3zi5s5jw
```

Comment:
This is not an example, because in the example, you have DID, while you should have a base64url-encoded proof.